### DISCUSSION TITLE
Cover-Enforcer CLI Tool

### DISCUSSION THREAD

**Discussion by rockymtngeek** [crocodilestick/Calibre-Web-Automated#157] [General]
I am not not that familiar with various cli commands and following the setup guide to run: cover-enforcer -all command from within the container (RECOMMENDED WITH FIRST TIME USE), which fails to do anything, it only returns:
sh: 2: cover-enforcer: not found
Can someone guide me on how thi command needs to be run? I have this is installed in Docker on Unraid. The cwa-check command also fails so I think I need something else before just these commands other than just the #
Looking to start using the app but don't want to until I have this and any other suggested settings done first. -thanks

Update - I do see an option in the gui settings called Generate Thumbnails - is that the same function?

**Comment by hestepige123** [comment]
Experiencing the exact same issue.

**Reply by hestepige123** [reply]
Figured it out - the culprit was using sh instead of bash.

I was doing this after following a tutorial:
`sudo docker exec -it [container-name] sh`

SSH into your NAS then run:
`sudo docker exec -it [container-name] bash`

Then `cover-enforcer -all` should run fine.

**Comment by Tooelite** [comment]
It would be really nice to have a terminal window inside the gui for output and a button to start the cover-enforcer via the gui.

Using these docker commands does not fit in so nicely,but I am glad the functionality is there.
